pkg/auth/dependency/sso/utils.go

```diff
@@ -1,35 +1,52 @@
 package sso
 
 import (
+	"time"
+
 	jwt "github.com/dgrijalva/jwt-go"
 
 	"github.com/skygeario/skygear-server/pkg/core/errors"
 )
 
-// CustomClaims is the type for jwt encoded
-type CustomClaims struct {
+type StateClaims struct {
 	State
 	jwt.StandardClaims
 }
 
-// NewState constructs a new state
-func NewState(params GetURLParams) State {
-	return params.State
+const jwtAudience = "skygear-server"
```

Comment:
The audience of the JWT should be the specific app, not all apps. We should use app ID/name instead.